core/cache/stringswatcher_test.go

```diff
@@ -1,113 +0,0 @@
-// Copyright 2018 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package cache_test
```

Comment:
Why not just change the package rather than moving the contents?

Reply:
Yes, I could have done that.